.gitignore

```diff
@@ -37,3 +37,6 @@ vignettes/*.pdf
 
 # R Environment Variables
 .Renviron
+
+# Access Databases
+*.mdb
```

Comment:
could add *.accdb if thinking that you won't swap everything to sqlite?

Reply:
ignoring the crc and fram databases, they're all mdb